frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -108,11 +206,29 @@ pods:
         {{/TASKCFG_ALL_SECURITY_TRANSPORT_ENCRYPTION_ENABLED}}
       repair:
         goal: ONCE
-        cmd: export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/) && ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/nodetool repair -pr -p ${JMX_PORT} -- $CASSANDRA_KEYSPACE
+        cmd: |
+              export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/)
+              {{#TASKCFG_ALL_SECURE_JMX_ENABLED}}
+              export HOME=$MESOS_SANDBOX
+              user=$(sed -nr '0,/re/s/.*(^)(.+) readwrite.*/\2/p' $MESOS_SANDBOX/jmx/access_file)
```

Comment:
works for 
```
controlRole readwrite
monitorRole readonly
```
but not for 
```
monitorRole readonly
controlRole readwrite
```
and both are valid access file content

Reply:
Thanks @zmalik , I will update it.

Reply:
updated sed command.